docs/docs/integrations/providers/comet_tracking.ipynb

```diff
@@ -4,95 +4,64 @@
    "cell_type": "markdown",
```

Comment:
Let's add a lot more information in this notebook (assuming it is going to be the main one), we should include:

1. Better information on getting an API key + link to self-hosting guide if you want to self host
2. More examples and link back to Opik, see the DeepEval docs for examples: https://python.langchain.com/docs/integrations/callbacks/confident/
3. Document both adding the callback as back of the `with_config` method and the `run` method
4. Document additional options like adding tags and metadata
5. Add more information about retrieving logged chains using `created_traces`

You can view the existing Opik docs for a more comprehensive LangChain documentation page: https://www.comet.com/docs/opik/tracing/integrations/langchain

Reply:
@jverre addressed these in my latest commit!